platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectProtocol.java

```diff
@@ -115,25 +115,25 @@ public boolean shouldAccept() {
         // released by the ReconnectTeacher (or by this component if we don't fail first).
```

Comment:
I think this comment needs to be updated. It is no longer released by ReconnectTeacher, but release by this class.

Reply:
Removed out of date comment.